crates/bevy_app/src/app.rs

```diff
@@ -749,6 +749,14 @@ impl App {
         self
     }
 
+    /// Check that a plugin has already been added to the app.
+    pub fn is_plugin_added<T>(&self) -> bool
```

Comment:
I find the naming a bit confusing here. Not every plugin is added to the hashmap. At the moment it also contains plugin groups.

Maybe `is_unique_plugin_added` would fit better?